### PR TITLE
add doc for strict login checking required for disabling login with LDAP email address

### PR DESCRIPTION
**Pull request by mrow4a**
Closes https://github.com/owncloud/docs/issues/2711

### CODE REVIEW

File: modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc [hunk 1]
```diff
@@ -235,6 +235,8 @@ LDAP Username::
 LDAP Email Address::
   If this value is checked, the login value will be compared to an email address in the LDAP directory; specifically, the `mailPrimaryAddress` and `mail` attributes.
 
+  IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective: **sudo -u www-data php occ config:system:set --type boolean --value true strict_login_enforced**
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc:238]:
```suggestion
IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective:
  
[source,console,subs="attributes+"]
----
{occ-command-example-prefix} config:system:set --type boolean --value true strict_login_enforced
----
```

**Comment by mmattel** [comment]
pls note to which branch a backport is needed (or not at all)

**Comment by mrow4a** [comment]
@mmattel 10.5

File: modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc [hunk 2]
```diff
@@ -235,7 +235,12 @@ LDAP Username::
 LDAP Email Address::
   If this value is checked, the login value will be compared to an email address in the LDAP directory; specifically, the `mailPrimaryAddress` and `mail` attributes.
 
-  IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective: **sudo -u www-data php occ config:system:set --type boolean --value true strict_login_enforced**
+IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective:
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc:238]:
```suggestion
IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective: +
```

File: modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc [hunk 3]
```diff
@@ -235,7 +235,12 @@ LDAP Username::
...
+
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc:239]:
```suggestion
```

**Comment by mmattel** [comment]
I missed the "+" sign at the end of the line, fixed...
PR is fine, approving when green

File: modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc [hunk 4]
```diff
@@ -235,6 +235,13 @@ LDAP Username::
 LDAP Email Address::
   If this value is checked, the login value will be compared to an email address in the LDAP directory; specifically, the `mailPrimaryAddress` and `mail` attributes.
 
+IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective:
+
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc:239]: ```suggestion```

File: modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc [hunk 5]
```diff
@@ -235,6 +235,13 @@ LDAP Username::
 LDAP Email Address::
   If this value is checked, the login value will be compared to an email address in the LDAP directory; specifically, the `mailPrimaryAddress` and `mail` attributes.
 
+IMPORTANT: Disallowing login with LDAP Email Address requires enabling strict login checking to be effective: +
+
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/user/user_auth_ldap.adoc:239]: ```suggestion```

**Comment by mmattel** [comment]
@mrow4a today I learned that you can´t suggest to remove a line and easy commit this...
I locally fixed the stuff, compiled the doc (ok), squashed the commits and pushed it. Things are fine now.
When green, approval and ready to merge.

**Review by mmattel — APPROVED** [review]
LGTM 👍

**Review by phil-davis — APPROVED** [review]

**Pull request merged**